### ISSUE TITLE
Project review by Nicholai Mauritzson

### ISSUE THREAD

**Issue by nicholaimauritzson** [teokem/project-work-2018-albertasdvirnas#1]
_I have read through your report and have some thoughts based on the projects requirements given in the course https://github.com/mlund/jupyter-course and some of my own observations._

The only discrepancies from the requirements that I could find was the fact that your figures were quite small. _(I would recommend making them larger. 183 mm width for single column report was given in the requirements)_

The supplied environment did not seem to work for me. Once implemented I could not open jupyter in my browser. I manually installed some packages and it worked fine locally on my computer. I did find some broken/missing code however. Your external methods _init__.py file, line 92, missing parenthesis. _(maybe this would have worked fine if I would have used your your environment. However it is always recommend to use as little environment dependent code as possible. Worked fine after I fixed it.)_

Since this subject is outside of my field, it was quite hard to follow along, but the code was very well documented and the structure easy to follow. I like the interactive plot, even if it was a bit slow to update on my computer.

Thanks for a good project!

Regards,
Nicholai

**Comment by albertasdvirnas** [comment]
Thanks you for the review! A good point about a broken line, it should have been removed. I don't know why the notebook wouldn't open for you, though!

**Comment by nicholaimauritzson** [comment]
May just have been my environment and some weird dependency. I'm not to good with these things.
